insights.go

```diff
@@ -17,6 +19,10 @@ func unregisterInsights() error {
 	return cmd.Run()
 }
 
+// insightsIsRegistered tries to get information if system is registered
+// using insights-client, and it can also return error. Current implementation
+// of insights-client is suboptimal. For this reason getting status of
+// insights-client is horrible heuristics. Hopefully, it will change in the future.
```

Comment:
Doc comments should document the objective or purpose for the function as well as when to expect certain conditions from the return values. They typically should not include a summary of the implementation of the function. This makes the comment more resilient to changes within the function. For example, when insights-client does improve its ability to report status more accurately, this comment could become inaccurate before we get a chance to come in and change the comment.

(Note: I don't know if this suggestion accurately describes the return conditions; that should be verified before applying this suggestion.)

```suggestion
// insightsIsRegistered checks whether insights-client reports its
// status as registered or not. If the system is registered, `true` is
// returned, otherwise `false` is returned, and `error` is filled with
// an error value.
```